web/src/main/ui/app/components/flows-new/edit-flow/ui/new-step-dialog.component.html

```diff
@@ -20,7 +20,7 @@ <h1 mat-dialog-title>{{data.title}}</h1>
     </mat-radio-group>
   </div>
   <mat-form-field *ngIf="selectedSource === 'collection' && newStepForm.value.type != 'ingest'">
-    <mat-select id="step-source-collection" class="type-select" placeholder="Source Collection">
+    <mat-select id="step-source-collection" class="type-select" placeholder="Source Collection" formControlName="sourceQuery">
```

Comment:
Looks like this should be: formControlName="sourceCollection"

Reply:
From DHFPROD-2096: sourceCollection: UI needs to create query from sourceCollection and pass it through as sourceQuery